install_config/configuring_aws.adoc

```diff
@@ -195,10 +189,17 @@ been the case when no cloud provider was being used) to using the AWS
 this issue:
 
 .  Log in to the CLI as a cluster administrator.
+. Check existing node labels:
++
+[source, bash]
+----
+$ oc describe node <node_name> | grep -Poz '(?s)Labels.*\n.*(?=Taints)'
+----
 .  Delete the nodes:
 +
+[source, bash]
 ----
 $ oc delete node <node_name>
 ----
-.  On each node host, restart the *atomic-openshift-node* service.
+.  On each node host, restart the {product-title} service.
```

Comment:
@gaurav-nelson I think advantage of the service being in the command was that the reader would know what to do. Without it, i'd suggest adding an example command:

# systemctl restart atomic-openshift-node

Reply:
Actually, this should probably be conditionalized for Origin too. See below with a better example than what I used.

Reply:
Thanks @bfallonf I have updated the docs to include suggested example steps.